lightning/src/ln/channel.rs

```diff
@@ -4173,6 +4173,11 @@ impl<Signer: Sign> Channel<Signer> {
 	/// Also returns the list of payment_hashes for channels which we can safely fail backwards
 	/// immediately (others we will have to allow to time out).
 	pub fn force_shutdown(&mut self, should_broadcast: bool) -> (Option<(OutPoint, ChannelMonitorUpdate)>, Vec<(HTLCSource, PaymentHash)>) {
+		// Note that we MUST NOT generate a monitor update which does anything but indicate
+		// force-closure - we're called during initialization prior to the chain_monitor in the
```

Comment:
Could this be stated more plainly as:

> Note that we MUST only generate a monitor update that indicates force-closure

The current phrasing is difficult to parse.